packages/twenty-front/src/modules/object-record/record-field/meta-types/input/components/NumberFieldInput.tsx

```diff
@@ -28,27 +28,30 @@ export const NumberFieldInput = ({
     persistNumberField,
   } = useNumberField();
 
+  const isPercentage =
```

Comment:
no need to pass isPercentage from there, it can be directly read from persistNumberField